function_schema.yml

```diff
@@ -0,0 +1,88 @@
+---
+$schema: https://json-schema.org/draft/2020-12/schema
+$id: https://jmespath.net/function_schema.yml
```

Comment:
Minor nit: should this be `jmespath.site` here as part of the _namespace_ ?

Reply:
YEP, I have no idea why I went with .net